CHANGES.rst

```diff
@@ -1,10 +1,118 @@
 .. _gammapy_0p14_release:
 
-0.14 (unreleased)
------------------
+0.14 (Sep 30, 2019)
+-------------------
+
+Summary
++++++++
+
+- Released on September 30th, 2019
+- 8 contributors
+- 95 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.14 features a new high level analysis interface. Starting from
+a YAML configuration file, it supports the standard use-cases of joint
+or stacked 3D as well as 1D reflected region analyses. It also supports
+computation of flux points for all cases. The usage of this new ``Analysis``
+class is demonstrated in the ``hess.ipynb`` tutorial.
+
+Following the proposal in :ref:`pig-016` the subpackages ``gammapy.background``
+and ``gammapy.image`` were removed. Existing functionality was moved to the
+``gammapy.cube`` and ``gammapy.spectrum`` subpackages.
+
+A new subpackge ``gammapy.modeling`` subpackage as introduced. All spectral,
+spatial, temporal and combined models were moved to the new namespace and
+renamed following a consistent naming scheme. This provides a much clearer
+structure of the model types and hierarchy for users.
+
+A new ``ConstantTemporalModel`` was introduced.
```

Comment:
@adonath - I didn't mean a separate paragraph for `ConstantTemporalModel`, just a separate sentence. It's currently not useful for users in any way, I would suggest to either remove it's mention, or to put this sentence at the end of the next paragraph after describing the changes in spatial models.

Reply:
Fixed